hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/snapshot/SnapshotDiffResponse.java

```diff
@@ -43,16 +45,59 @@ public static JobStatus fromProtobuf(JobStatusProto jobStatusProto) {
     }
   }
 
+  /**
+   * Snapshot diff cancel status enum.
+   */
+  public enum CancelStatus {
+    JOB_NOT_CANCELED("Job is not canceled"),
```

Comment:
IIUC this indicates "Job has never been cancelled"?

Reply:
Yes, I'll change the description to `Job hasn't been cancelled` to make it more explanatory.